products/rhel8/profiles/stig.profile

```diff
@@ -529,6 +529,7 @@ selections:
 
     # RHEL-08-020040
     - configure_tmux_lock_command
+    - configure_tmux_lock_keybinding
```

Comment:
I think you will have to update the profile stability tests

Reply:
Added by rebase.